commands.yaml

```diff
@@ -18,21 +18,21 @@
   check_command: |
     defaults read /Library/Preferences/com.apple.Bluetooth ControllerPowerState | grep 0
   fix_command: |
-    sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0 &&  sudo killall -HUP blued
+    sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0; sudo killall -HUP blued
   enabled: true
 
 
 - title: "Disable infrared receiver"
   check_command: |
     defaults read /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled | grep 0
   fix_command: |
-    defaults write /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled -int 0
+    sudo defaults write /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled -int 0
   enabled: true
 
 
 - title: "Disable AirDrop"
   check_command: |
-    defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1
+    sudo defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1
```

Comment:
> Why are you adding `sudo` here?

Hmm, now this works without sudo.

> I might just make some sort of "is root" check at the top of the program and get rid of all these.

I agree with all of you, root/sudo check is better.

> But I see you merged that just after you wrote your comment.

I think it's because of two new commands and two real sudo "bugs". (commit 4c9a0a7) Future sudo/root check will cleanup all unnecessary stuff.

Thanks for your work, Scott!